webapp/portlet/src/main/webapp/dlp-quarantine/dlpAdministrationServices.js

```diff
@@ -100,9 +100,9 @@ export function checkIsAdminMemberGroup() {
   });
 }
 
-export async function getPermissionsData(query) {
+export async function getPermissionsData(query, offset, limit) {
   try {
-    const response = await fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/groups?q=${query}`, {
+    const response = await fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/dlp/items/groups?q=${query}&offset=${offset || 0}&limit=${limit|| 0}`, {
```

Comment:
Why it was not possible to use default API ? What are the limitation which lead to create and use a custom rest service ?

Reply:
To suggest only groups in the permissions drawer, the default rest services suggest spaces and groups.

Reply:
As explained in the task, this task is paused, because this part will be reworked and simplified. So please : do not add a new rest function in dlp/items/groups to not add deadly code, and keep using /v1/groups for this moment.
Thakns